packages/core/constants/index.ts

```diff
@@ -31,6 +31,12 @@ export const TOP_LEVEL_FILE_ANNOTATIONS = [
         description: "Path to file in the cloud.",
         type: AnnotationType.STRING,
     }),
+    new Annotation({
```

Comment:
Looks like this is the pattern for dealing with file properties?

Reply:
Not sure what the comment is here but my interpretation is this format allows us to create our own annotations in BFF which we later propagate with the file properties.